packages/clis/generator/index.ts

```diff
@@ -374,6 +423,95 @@ function handleFootprintsCommand(
   logger.success('done.');
 }
 
+function handleContoursCommand(
+  args: ReturnType<typeof contoursCommandBuilder>,
+) {
+  const toJsonPath = (map: 'usa' | 'europe', suffix: string) =>
+    path.join(args.inputDir, `${map}-${suffix}.json`);
+
+  for (const map of args.map) {
+    logger.log('calculating', map, 'contours');
+    const points = readArrayFile<[number, number, number]>(
+      toJsonPath(map, 'elevation'),
+    );
+    const geoJson = convertToContoursGeoJson({ map, points });
+    if (args.dryRun) {
+      continue;
+    }
+
+    let geoJsonPath: string | undefined;
+    const gamePrefix = map === 'usa' ? 'ats' : 'ets2';
+    const filename = `${gamePrefix}-contours.geojson`;
+    if (args.type.includes('geojson')) {
+      geoJsonPath = path.join(args.outputDir, filename);
+      logger.log('writing', geoJsonPath + '...');
+      writeGeojsonFile(geoJsonPath, geoJson);
+    }
+    if (!args.type.includes('pmtiles')) {
+      continue;
+    }
+
+    let cleanupGeoJson = false;
+    if (geoJsonPath == null) {
+      //geoJsonPath = path.join(os.tmpdir(), filename);
+      geoJsonPath = path.join(args.outputDir, filename);
+      logger.log('writing temporary GeoJSON file...');
+      writeGeojsonFile(geoJsonPath, geoJson);
+      cleanupGeoJson = true;
+    }
+
+    const minAttributes = ['elevation'];
```

Comment:
Reminder to update [`Contours`](https://github.com/truckermudgeon/maps/pull/7/files#diff-dcc4891cb441c22f76dc3402c34e9173c0cc8913cc0d6f84999cba4c86f042ccR16) component to reference `elevation` (absolute values) instead of `index` (relative values)